aws-operator-role/iam-giantswarm.tf

```diff
@@ -1,16 +1,17 @@
-### AWS Operator Role ###
-# Get latest policy
+locals {
+  operator_role_name = "GiantSwarmAWSOperator"
```

Comment:
I checked and only `gauss` has different role name for TC account. I'll recreate the role there.

Reply:
This is sorted.